scripts/install_third_party_libs.py

```diff
@@ -32,6 +32,8 @@
     ('pyyaml', '5.1.2', os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')),
     ('future', '0.18.2', os.path.join(
         'third_party', 'python_libs')),
+    ('certifi', '2020.12.5', os.path.join(
+        TOOLS_DIR, 'certifi-2020.12.5')),
```

Comment:
Why do we need to install it here but also in requirements?

Reply:
I've removed it from requirements